hack/deploy-hub-local/create-vm.yml

```diff
@@ -1,6 +1,7 @@
 parameters:
   nodes: 3
-  clusters: 2 
+  clusters: 2
+  singlenic: true
```

Comment:
same

Reply:
that's the idea...Talked with Flavio, the default will be one nic